docs/development/priority-classes.md

```diff
@@ -25,7 +25,7 @@ When using the `gardener-operator` for managing the garden runtime and virtual c
 | `gardener-garden-system-500`      | 999999500 | `virtual-garden-etcd-events`, `virtual-garden-etcd-main`, `virtual-garden-kube-apiserver`, `gardener-apiserver` |
 | `gardener-garden-system-400`      | 999999400 | `virtual-garden-gardener-resource-manager`                                                                      |
 | `gardener-garden-system-300`      | 999999300 | `virtual-garden-kube-controller-manager`, `vpa-admission-controller`, `etcd-druid`, `nginx-ingress-controller`  |
-| `gardener-garden-system-200`      | 999999200 | `vpa-recommender`, `vpa-updater`, `hvpa-controller`                                                             |
+| `gardener-garden-system-200`      | 999999200 | `vpa-recommender`, `vpa-updater`, `hvpa-controller`, `fluent-operator`, `fluent-bit`, `vali`                                                            |
```

Comment:
Please move to lowest priority in the list, .i.e. `gardener-garden-system-100` (as done for seed).